raiden/tests/scenarios/ci/sp2/pfs1_get_a_simple_path.yaml

```diff
@@ -45,12 +45,6 @@ scenario:
             - open_channel: {from: 0, to: 1, total_deposit: 1_000_000_000_000_000_000}
             - open_channel: {from: 1, to: 2, total_deposit: 1_000_000_000_000_000_000}
             - open_channel: {from: 2, to: 3, total_deposit: 1_000_000_000_000_000_000}
-      - parallel:
-          name: "Deposit in the other directions"
-          tasks:
-            - deposit: {from: 1, to: 0, total_deposit: 1_000_000_000_000_000_000}
-            - deposit: {from: 2, to: 1, total_deposit: 1_000_000_000_000_000_000}
-            - deposit: {from: 3, to: 2, total_deposit: 1_000_000_000_000_000_000}
```

Comment:
Why was this removed? 

THis causes https://github.com/raiden-network/raiden/issues/5460